x-pack/legacy/plugins/searchprofiler/public/np_ready/application/containers/profile_query_editor.tsx

```diff
@@ -86,8 +86,10 @@ export const ProfileQueryEditor = memo(() => {
                 <EuiFieldText
                   disabled={!licenseEnabled}
                   inputRef={ref => {
-                    indexInputRef.current = ref!;
-                    ref!.value = DEFAULT_INDEX_VALUE;
+                    if (ref) {
+                      indexInputRef.current = ref!;
+                      ref!.value = DEFAULT_INDEX_VALUE;
```

Comment:
Do we still need the exclamation mark now that it is wrapped in the `if` statement?